edi_sale_edifact_oca/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1 @@
+This module provides a demo exchange type setting for EDIFACT standard.
```

Comment:
if it's only for demo/example purpose, we should rename this module :wink:

Reply:
hehe it is for demo/example purpose but also includes tests and might one day include additional specific components for EDIFACT if needed